### ISSUE TITLE
Floating npcs, missing tomes and stuff 

### ISSUE THREAD

**Issue by psychadelicc** [wabbajack-tools/ussedp#14]
So a few weeks ago, when this mod was new, I used this patcher and everything is working fine. All my mods are working fine. All the mods i will mention that have some bugs were working until this happened

Steam suddenly out of nowhere updated skyrim so I had to redo all the steps. I updated skyrim, verified integrity of game files, and used this patcher. Everything is working fine now BUT, there are bugs that were nonexistent before. With skyrim at war, all npcs are floating when this never happened before. With realm of lorkhan, the spell books that were supposed to be near the winterhold portal are gone. When u walk inside whiterun for the firstime, theres this battle born talking to a non existent Adrianne Avenicci until she walked out of the blacksmite building a few seconds later. NONE of this ever happened until today when skyrim updated, then followed all the steps for the patcher.

**Comment by psychadelicc** [comment]
Regarding floating npcs, I also have actor limit fix

>

**Comment by psychadelicc** [comment]
Alright I was able to fix it last night before I went to sleep but im not sure if everthing is fixed. I fixed it by redownloading mods that requires some files to be put in the skyrim root folder. SSE engine fixes [three dll files]  , DLLpluginloader [ the binkw64.dll ], d3dcompiler_47 [ idk which mod requires this but I just saw it ].

I didnt see that in the description, as of 1.3 it changes the dll files so does it only change the original dll files? because as far as I can remember, the dll files SSE engine fixes adds are new dll files.
